packages/device/.mocharc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What does this file do?

Reply:
It is needed so that we can run the mocha tests directly as typescript files.